webcompat/templates/index.html

```diff
@@ -1,13 +1,13 @@
 {%- extends "layout.html" -%}
 {%- block body -%}
 {% include "topbar.html" %}
-<main role="main" class="wc-HomePage">
-  <div class="wc-Loader js-Loader"></div>
-  {% include "home-page/hero.html" %}
+<main role="main">
+<!--   <div class="wc-Loader js-Loader"></div>
```

Comment:
Is this temporary, or should we just delete it?

Reply:
I'd like to leave it in for now as this is related to https://github.com/webcompat/webcompat.com/issues/1529.

Reply:
ok, let's be sure to not let that slip to production please. commented out code is super confusing. (and it can just as easily be pointed to with a URL in the related issue)

Reply:
+1